src/lib.rs

```diff
@@ -392,12 +422,12 @@ pub fn info_wallet(chain_type: &ChainTypes, binary_path: &str, password: &str) -
 pub fn confirm_transaction(chain_type: &ChainTypes, binary_path: &str, password: &str) {
     // time dependence
     let t0 = Instant::now();
-    let two_minute = Duration::from_secs(120);
+    let n_minute = Duration::from_secs(5);
```

Comment:
I wanted to put 5 minutes and ended up forgetting to put `5*60` it was only 5 seconds instead of 5 minutes.

n is meant to be a variable measure of minute, instead of being two it can be any quantity